hocr-combine

```diff
@@ -12,24 +13,15 @@ from lxml import etree, html
 ### main program
 ################################################################
 
-def print_usage():
-    sys.stderr.write("combine multiple hOCR documents into one\n\n")
-    sys.stderr.write("usage: %s file1.html file2.html...\n"%sys.argv[0])
-
-if len(sys.argv)>1 and (sys.argv[1] == '-h' or sys.argv[1] == '--help'):
-    print_usage()
-    sys.exit(0)
-
-if len(sys.argv)<2:
-    print_usage()
-    sys.exit(1)
-
-doc = html.parse(sys.argv[1])
+parser = argparse.ArgumentParser(description="combine multiple hOCR documents into one")
+parser.add_argument("files", help="hOCR files", type=argparse.FileType('r'),  nargs='+', default=sys.stdin)
```

Comment:
This works with a single file as well, also it will then just output that file again, i.e. no need to force at least 2 here (which is non-trivial with argparse).